zipkin-lens/src/components/DiscoverPage/DiscoverPageContent.tsx

```diff
@@ -699,13 +699,19 @@ const DiscoverPageContent: React.FC<DiscoverPageContentProps> = ({
 
 export default DiscoverPageContent;
 
-const LookbackButton = styled(Button).attrs<{ isShowingLookbackMenu: boolean }>(
-  ({ isShowingLookbackMenu }) => ({
-    variant: 'outlined',
-    startIcon: <FontAwesomeIcon icon={faHistory} />,
-    endIcon: isShowingLookbackMenu ? <ExpandLessIcon /> : <ExpandMoreIcon />,
-  }),
-)<{ isShowingLookbackMenu: boolean }>`
+const LookbackButton = styled(
+  ({
+    isShowingLookbackMenu,
+    ...rest
+  }: { isShowingLookbackMenu: boolean } & ButtonProps) => (
+    <Button
+      variant="outlined"
+      startIcon={<FontAwesomeIcon icon={faHistory} />}
+      endIcon={isShowingLookbackMenu ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+      {...rest}
+    />
+  ),
+)`
```

Comment:
This change is intended to avoid this error...

![スクリーンショット 2020-09-06 15 40 23](https://user-images.githubusercontent.com/19551419/92319834-565d5680-f057-11ea-9f89-319654a5a329.png)

Reply:
Yeah styled-components is for CSS, not material-ui props :)